airbyte-integrations/connectors/source-genesys/integration_tests/sample_state.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can you add a sample state here? It will be useful for the integration tests. A good reference is here: 

https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-github/integration_tests/sample_state.json

Reply:
@YiyangLi using all full refresh streams for now only, no incremental/date based streams. Will remove the file